kitsune/search/es_utils.py

```diff
@@ -94,9 +92,36 @@ def es_client(**kwargs):
     """Return an ES Elasticsearch client"""
     # prefer a cloud_id if available
     if es_cloud_id := settings.ES_CLOUD_ID:
-        kwargs.update({"cloud_id": es_cloud_id, "http_auth": settings.ES_HTTP_AUTH})
+        kwargs.update({"cloud_id": es_cloud_id, "basic_auth": settings.ES_HTTP_AUTH})
     else:
-        kwargs.update({"hosts": settings.ES_URLS})
+        # Elasticsearch 8.x settings
+        es_settings = {
+            "hosts": settings.ES_URLS,
+            "request_timeout": settings.ES_TIMEOUT,
+            "retry_on_timeout": settings.ES_RETRY_ON_TIMEOUT,
+            # SSL settings - these are needed for ES8 which requires SSL by default
+            "verify_certs": settings.ES_VERIFY_CERTS,
```

Comment:
what's the reason for introducing all these new settings?

Reply:
ES8 requires the SSL settings. `retry_on_timeout` was an attempt to improve reliability when troubleshooting test issues; the same with the `sniff_on_start` and `sniff_on_connection_fail`.  
We can drop the `sniff_*` settings, but should consider leaving the `retry_on_timeout`. May not be critical but it could help when/if there are real connection issues.